elements/pl-hidden-hints/pl-hidden-hints.py

```diff
@@ -0,0 +1,102 @@
+import chevron
+import lxml.html
+import prairielearn as pl
+from typing import TypedDict, Tuple, List
+
+
+class HintsDict(TypedDict):
+    """A class with type signatures for the partial scores dict"""
+    html_hints: List[Tuple[int, str]]
+    submission_count: int
+
+
+def prepare(element_html: str, data: pl.QuestionData) -> None:
+    # Set submit count
```

Comment:
This comment doesn't line up with what's on line 15 - maybe move down to line 18?